src/lib.rs

```diff
@@ -377,6 +486,16 @@ pub trait Miniconf<const Y: usize = 1> {
     ///
     /// See also [Miniconf::iter_paths].
     ///
+    /// ```
+    /// # use miniconf::{Miniconf, Metadata};
+    /// # use heapless::String;
+    /// #[derive(Miniconf)]
+    /// struct S {foo: u32};
```

Comment:
Gotta remove teh `heapless::String` throughout or fix the import

Reply:
It's both needed and correct. The problem is elsewhere.